frontend/components/TextInput/TextInput.jsx

```diff
@@ -104,21 +126,35 @@ export default class TextInput extends Component {
     this.state.value = props.value || ''
   }
 
+  componentDidMount() {
+    this.adjustHeightIfNeeded()
+  }
+
   render() {
     const {
       className,
+      heightType,
       id,
       inLabel,
       placeholder,
       readonly,
       required,
+      resizable,
+      rows,
       type,
       value
     } = this.props
 
     let inputStyle = new Style(styles, 'input')
 
     inputStyle.addIf('input-in-label', inLabel)
+    inputStyle.add(resizable ? `resizable` : `not-resizable`)
```

Comment:
Any reason to use template literals in place of regular strings?

Reply:
I'm used to it. No other reason. It's also less work to add templates if I need to, so I've just started to default to it. I'm all for not doing so if that's the rules / we're using `standard` / etc. Let's eslint the lot, really.